src/InputManager.h

```diff
@@ -7,6 +7,7 @@
 
 class InputManager {
 
+    const int MAX_PLAYERS = 4; // Maximum number of players in a game
```

Comment:
Usually I find these belong best in anonymous spaces where they're being used. So in the source file:

```
namespace {
    const int MAX_PLAYER = 4;
}
```

Reply:
Sure. Would prefer to keep these game-wide constants together though once we find the need for something like that